### PR TITLE
Object Pair Constraints

### PR DESCRIPTION
**Pull request by RobertElliottKrieter**
Criticism gladly accepted. My apologies if formatting is rough-- still getting accustomed to GitHub.

### CODE REVIEW

File: Assignment Part One: Object-Property Pai.html — Logic-for-Ontologists-Fall-2024 [Codespaces: supreme.html [hunk 1]
```diff
@@ -1,145 +1,114 @@
...
+Assignment Part One: Object-Property Pair contradictions.
+
+"PROVIDE OWL-2 DIRECT SEMANTIC BASED EXPLANATIONS
+FOR UNSATISFIABILITY AND INCONSISTENCY RESULTS
+STEMMING FROM ROLE CONSTRAINT PAIRS."
+
+1.) FUNCTIONAL COMBINED WITH TRANSITIVE BECOMES UNDECIDEABLE
+
+Functional object property denotes cardinality of (1) such relationship between 2 instances.
+For example, John (x) is connected to Mary (y) by object-property "has father" is unique.
```
Comment (RobertElliottKrieter) [inline Assignment Part One: Object-Property Pai.html — Logic-for-Ontologists-Fall-2024 [Codespaces: supreme.html:10]: Switch Mary with Steve or vis a versa. For every (y) related to (x) there can only be one (x) who hold this property relation

Comment (RobertElliottKrieter) [reply Assignment Part One: Object-Property Pai.html — Logic-for-Ontologists-Fall-2024 [Codespaces: supreme.html:10]: Please see "Initial Work..." file that does not have messed up code of the left hand side for commenting.

**Comment by RobertElliottKrieter** [comment]
https://github.com/elenamiliv/Logic-for-Ontologists-Fall-2024/pull/3

File: Assignment Part One: Object-Property Pai.html — Logic-for-Ontologists-Fall-2024 [Codespaces: supreme.html [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+Functional object property denotes cardinality of (1) such relationship between 2 instances.
+For example, John (x) is connected to Steve (y) by object-property "has father" is unique.
+That is, Steve (y) can only bear the object-property relationship "has_father" with one other individual, namely, John (x).
+
+Transitive object-property goes according to the following:
+(x) relates to (y) by object-property 1 and (y) related to (z) by same object-property 1, 
+So (x) also relates to (z) by virture of object-property 1.
+x --> y, y --> z Therefore x --> z
+
+When running a reasoner the combination of these becomes undecideable because a functional property must only be only be obtained between one individual and another,
```
Comment (johnbeve) [inline Assignment Part One: Object-Property Pai.html — Logic-for-Ontologists-Fall-2024 [Codespaces: supreme.html:18]: I need more precision in your explanation. I encourage you use variables x, y etc to make clear targets of object properties, among other things.

File: Assignment Part One: Object-Property Pai.html — Logic-for-Ontologists-Fall-2024 [Codespaces: supreme.html [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+however, if it is applied transitively it will carry over to subsequent pairs who are also connected by this same object property.
+
+
+2.) INVERSE-FUNCTIONALITY AND TRANSITIVITY BECOMES UNDECIDEABLE
+
+Inverse functionality provides that each individual paired with another with this relation can have at most one incoming connection of said object property.
+Ex: Inverse_Functional_Property "father of";
+    Father_of (Joey, Richard)      = "Joey is father_of Richard"
```
Comment (johnbeve) [inline Assignment Part One: Object-Property Pai.html — Logic-for-Ontologists-Fall-2024 [Codespaces: supreme.html:26]: Why not use variables here too for clarity?

File: Assignment Part One: Object-Property Pai.html — Logic-for-Ontologists-Fall-2024 [Codespaces: supreme.html [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+--> This means that the instance, Richard, can only be inverse-functionally related to Joey and no other instance because any person can only have one father.
+--> Conversely the object-property "father_of" may be obtained between Joey and other instances, who will only have this relation which Joey. 
+
+Transitive object-property goes according to the following:
+(x) relates to (y) by object-property 1 and (y) related to (z) by same object-property 1, 
+So (x) also relates to (z) by virture of object-property 1.
+x --> y, y --> z Therefore x --> z
+
+When running a reasoner the combination of these becomes undecideable because an inverse functional property holds that an object can be connected to at most one other object through such a property relation,
+however, if it is applied transitively it whill carry over to subsequent pairs who are also connected by this same object property.
```
Comment (johnbeve) [inline Assignment Part One: Object-Property Pai.html — Logic-for-Ontologists-Fall-2024 [Codespaces: supreme.html:36]: I do not understand what you are saying here

**Review by johnbeve — CHANGES_REQUESTED** [review]
You only did what I consider to be the easy part of the project, and you didn't complete it. 
If you would like credit for this project, you will need to complete all of the work, have it reviewed by your peers, then submit again.

**Pull request closed without merging**